modules/op-release-notes-1-16.adoc

```diff
@@ -0,0 +1,603 @@
+    name: default-lokistack
+    tls:
+      ca:
+        configMapName: openshift-service-ca.crt
+        key: service-ca.crt
+    type: lokiStack
+  pipelines:
+  - inputRefs:
+    - only-tekton
+    name: default-logstore
+    outputRefs:
+    - default-lokistack
+  serviceAccount:
+    name: collector
+----
++
+.YAML manifest for the `ClusterLogForwarder` CR if you installed OpenShift Logging version 5
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: ClusterLogForwarder
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  inputs:
+  - name: only-tekton
+    application:
+      selector:
+        matchLabels:
+          app.kubernetes.io/managed-by: tekton-pipelines
+  pipelines:
+    - name: enable-default-log-store
+      inputRefs: [ only-tekton ]
+      outputRefs: [ default ]
+----
++
+Finally, in the `TektonResult` CR in the `openshift-pipelines` namespace, set the following additional parameters:
++
+--
+* `loki_stack_name`: The name of the `LokiStack` CR, typically `logging-loki`.
+* `loki_stack_namespace`: The name of the namespace where LokiStack is deployed, typically `openshift-logging`.
+--
++
+.Example of configuring LokiStack log forwarding in the `TektonResult` CR
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonResult
+metadata:
+  name: result
+spec:
+  targetNamespace: tekton-pipelines
+# ...
+  loki_stack_name: logging-loki
+  loki_stack_namespace: openshift-logging
+----
+
+[id="breaking-changes-1-16_{context}"]
+== Breaking changes
+
+* With this update, the metric name for the `EventListener` object for pipelines triggers that counts received events is changed from `eventlistener_event_count` to `eventlistener_event_received_count`.
+
+[id="known-issues-1-16_{context}"]
+== Known issues
+
+* The `jib-maven` `ClusterTask` does not work if you are using {OCP} version 4.16 and later.
+
+[id="fixed-issues-1-16_{context}"]
+== Fixed issues
+
+* Before this update, when you uninstalled {tekton-hub} by deleting the `TektonHub` CR, the pod of the `hub-db-migration` job was not deleted. With this update, uninstalling {tekton-hub} deletes the pod.
+
+* Before this update, when you used {tekton-results} to store pod logs from pipelines and tasks, the operation to store the logs sometimes failed. The logs would include the `UpdateLog` action failing with the `canceled context` error. With this update, the operation completes correctly. 
+
+* Before this update, when you passed a parameter value to a pipeline or task and this value included more than one variable, and they use both full and short reference formats, for example, `$(tasks.task-name.results.variable1) + $(variable2)`, {pipelines-shortname} did not interpret the value correctly; the pipeline run or task run could stop execution and the Pipelines controller could crash. With this update, {pipelines-shortname} interprets the value correctly and the pipeline run or task run completes.
```

Comment:
```suggestion
* Before this update, passing a parameter value to a pipeline or task where the value included more than one variable and both full and short reference formats, {pipelines-shortname} did not interpret the value correctly. For example, `$(tasks.task-name.results.variable1) + $(variable2)`. The pipeline run or task run could stop execution and the Pipelines controller could crash. With this update, {pipelines-shortname} interprets the value correctly and the pipeline run or task run completes.
```

Reply:
Suggested reworking eliminating some pronouns and adding the example to a separate sentence instead of interjecting it.

Reply:
I don't really think this change works: 

- "passing a parameter value... {pipelines-shortname} did not interpret the value correctly" sounds like it's Pipelines that passed the value, but in reality the user did.

- The example looks like a partial sentence disconnected from the previous sentence.

I do agree that instead of the semicolon there should be a full stop before "The pipeline run or task run could stop".

Reply:
@mramendi I aggree. I, however, think we could make this part better: `...and this value included more than one variable, and they use both full and short reference formats, ...`

I think this would work too and be more concise: `...and the value included more than one variable with both full and short reference formats...` WDYT?